packages/react/package.json

```diff
@@ -26,27 +26,47 @@
     "url": "https://opencollective.com/mui-org"
   },
   "exports": {
-    ".": {
-      "import": "./src/index.ts"
-    },
-    "./utils": {
-      "import": "./src/utils/index.ts"
-    },
-    "./*": {
-      "import": "./src/*/index.ts"
-    }
+    ".": "./src/index.ts",
+    "./accordion": "./src/accordion/index.ts",
+    "./alert-dialog": "./src/alert-dialog/index.ts",
+    "./checkbox": "./src/checkbox/index.ts",
+    "./checkbox-group": "./src/checkbox-group/index.ts",
+    "./collapsible": "./src/collapsible/index.ts",
+    "./dialog": "./src/dialog/index.ts",
+    "./field": "./src/field/index.ts",
+    "./fieldset": "./src/fieldset/index.ts",
+    "./form": "./src/form/index.ts",
+    "./menu": "./src/menu/index.ts",
+    "./number-field": "./src/number-field/index.ts",
+    "./popover": "./src/popover/index.ts",
+    "./preview-card": "./src/preview-card/index.ts",
+    "./progress": "./src/progress/index.ts",
+    "./radio": "./src/radio/index.ts",
+    "./radio-group": "./src/radio-group/index.ts",
+    "./scroll-area": "./src/scroll-area/index.ts",
+    "./select": "./src/select/index.ts",
+    "./separator": "./src/separator/index.ts",
+    "./slider": "./src/slider/index.ts",
+    "./switch": "./src/switch/index.ts",
+    "./tabs": "./src/tabs/index.ts",
+    "./text-input": "./src/text-input/index.ts",
+    "./tooltip": "./src/tooltip/index.ts",
+    "./unstable-no-ssr": "./src/unstable-no-ssr/index.ts",
+    "./unstable-use-media-query": "./src/unstable-use-media-query/index.ts",
+    "./utils": "./src/utils/index.ts"
```

Comment:
Added the unstable- prefix to NoSsr and useMediaQuery as they are not documented yet (and their API might change).